roles/system_packages/vars/main.yml

```diff
@@ -0,0 +1,88 @@
+---
+pkgs:
+  apparmor:
+    - "{{ ansible_os_family == 'Debian' }}"
+  apt-transport-https:
+    - "{{ ansible_os_family == 'Debian' }}"
+  aufs-tools:
+    - "{{ ansible_os_family == 'Debian' }}"
+    - "{{ ansible_distribution_major_version == '10' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  bash-completion: []
+  conntrack:
+    - "{{ ansible_os_family in ['Debian', 'RedHat'] }}"
+    - "{{ ansible_distribution != 'openEuler' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  conntrack-tools:
+    - "{{ ansible_os_family == 'Suse' or ansible_distribution in ['Amazon', 'openEuler'] }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  container-selinux:
+    - "{{ ansible_os_family == 'RedHat' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  curl: []
+  device-mapper:
+    - "{{ ansible_os_family == 'Suse' or ansible_distribution == 'openEuler' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  device-mapper-libs:
+    - "{{ ansible_os_family == 'RedHat' }}"
+    - "{{ ansible_distribution != 'openEuler' }}"
+  e2fsprogs: []
+  ebtables: []
+  gnupg:
+    - "{{ ansible_distribution == 'Debian' }}"
+    - "{{ ansible_distribution_major_version in ['11', '12'] }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  iproute:
+    - "{{ ansible_os_family == 'RedHat' }}"
+  iproute2:
+    - "{{ ansible_os_family != 'RedHat' }}"
+  ipset:
+    - "{{ kube_proxy_mode != 'ipvs' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  iptables:
+    - "{{ ansible_os_family in ['Debian', 'RedHat'] }}"
+  iputils:
+    - "{{ not ansible_os_family in ['Flatcar', 'Flatcar Container Linux by Kinvolk', 'Debian'] }}"
+    - "{{ main_access_ip is defined }}"
+    - "{{ ping_access_ip }}"
+    - "{{ not is_fedora_coreos }}"
+  iputils-ping:
+    - "{{ ansible_os_family == 'Debian' }}"
+    - "{{ main_access_ip is defined }}"
+    - "{{ ping_access_ip }}"
+  ipvsadm:
+    - "{{ kube_proxy_mode == 'ipvs' }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  libseccomp:
+    - "{{ ansible_os_family == 'RedHat' }}"
+  libseccomp2:
+    - "{{ ansible_os_family in ['Debian', 'Suse'] }}"
+    - "{{ 'k8s_cluster' in group_names }}"
+  libselinux-python:  # TODO: Handle rehat_family + major < 8
```

Comment:
Will this TODO be done? If we are on schedule for the next release, we will drop Ansible 9 (upgrade to 10 or higher), which will be unsupported on RHEL 8 systems.

Reply:
Hum, I saw it and I thought I had taken the occasion to remove it, since RHEL < 8 is already dropped ; it slipped through the cracks probably. I'll handle that ASAP

Reply:
I'm also not sure if Amazon is an extended RHEL8. In that case we'll probably have to drop it as well, but we'll see that after 2.28

Reply:
I think probably not. I launched an instance (the latest AMI 2) and checked the Python version; their Python version is 3.7.

According to Ansible Documentation: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix

<img width="789" alt="image" src="https://github.com/user-attachments/assets/227b9f6a-adf0-4280-97d3-50390f3ef4ba" />

I think it's no problem after 2.28.